step-capstone/src/components/TravelObjectForms/AddHotel.js

```diff
@@ -22,7 +22,7 @@ export default function AddHotel(props) {
     const [endDate, setEndDate] = useState(overwriting ? props.data.endDate : props.startDate);
     const [checked, setChecked] = useState(overwriting ? props.data.finalized : false);
     const [title, setTitle] = useState(overwriting ? props.data.title : "");
-    const [location, setLocation] = useState(overwriting ? { address: props.data.location, coordinates: props.data.coordinates } : { address: null, coordinates: null });
+    const [location, setLocation] = useState(overwriting ? { address: props.data.location, coordinates: props.data.coordinates, placeId: props.data.placeId} : null);
```

Comment:
nit: can placeId not be part of location object - it doesnt seem to really fit with the purely location components of the place like address and coordinates. here and in addEvent I believe.

Reply:
Right now, location is storing the result of the autocomplete, which is an object containing all three--do you want for me to return {placeID, {address, coordinates} instead? Also, they're stored separately in the database (I realized I forgot to do it in AddHotel, but it's there in useEffect in AddEvent and AddFlight), I just keep them together in the form state for easy manipulation when the autocomplete location is changed/selected.

Reply:
i think its fine as is since its all coming from the autocomplete and is stored seperately in the database. i just didnt want to have to do travelObject.location.placeID, but if its just in the form, its ok.